AppServer/google/appengine/api/taskqueue/taskqueue_distributed.py

```diff
@@ -84,6 +84,8 @@ class TaskQueueServiceStub(apiproxy_stub.APIProxyStub):
   This stub executes tasks when enabled by using the dev_appserver's AddEvent
   capability.
   """
+  _ACCEPTS_REQUEST_ID = True
```

Comment:
I didn't see this constant used in this PR: is that something internal in the SDK?